easy-digital-downloads.php

```diff
@@ -152,17 +160,18 @@ public static function instance() {
 			add_action( 'plugins_loaded', array( self::$instance, 'load_textdomain' ) );
 
 			self::$instance->includes();
-			self::$instance->roles         = new EDD_Roles();
-			self::$instance->fees          = new EDD_Fees();
-			self::$instance->api           = new EDD_API();
-			self::$instance->session       = new EDD_Session();
-			self::$instance->html          = new EDD_HTML_Elements();
-			self::$instance->emails        = new EDD_Emails();
-			self::$instance->email_tags    = new EDD_Email_Template_Tags();
-			self::$instance->customers     = new EDD_DB_Customers();
-			self::$instance->customer_meta = new EDD_DB_Customer_Meta();
-			self::$instance->payment_stats = new EDD_Payment_Stats();
-			self::$instance->cart          = new EDD_Cart();
+			self::$instance->roles           = new EDD_Roles();
+			self::$instance->fees            = new EDD_Fees();
+			self::$instance->api             = new EDD_API();
+			self::$instance->session         = new EDD_Session();
+			self::$instance->html            = new EDD_HTML_Elements();
+			self::$instance->emails          = new EDD_Emails();
+			self::$instance->email_tags      = new EDD_Email_Template_Tags();
+			self::$instance->customers       = new EDD_DB_Customers();
+			self::$instance->customer_meta   = new EDD_DB_Customer_Meta();
+			self::$instance->payment_stats   = new EDD_Payment_Stats();
+			self::$instance->cart            = new EDD_Cart();
+			self::$instance->extensionRegistry = new \EDD\Licensing\ExtensionRegistry();
```

Comment:
Since the lines above had spaces added to accommodate the original variable name, can we go ahead and add a couple more due to the new variable name?